assets/components/atoms/link/link.scss

```diff
@@ -1,27 +1,31 @@
 @charset 'utf-8';
 
-a:not([class*="btn"]) {
+a,
+a:hover,
+a:active { text-decoration: none; }
+
+a:not([class*="btn"]):not([class*="teaser"]):not([class*="card"]):not(.social-icon) {
   position: relative;
-  text-decoration: none;
   border-radius: 1px;
-  border-bottom: 1px solid $red;
   transition:
-    color 0.3s,
-    border-color 0.15s,
+    color 0.15s ease-in-out,
     box-shadow 0.15s;
 
-  &:not([class*="teaser"]):not([class*="card"]) {
-    @extend .focused;
-  }
+  @include pretty-link($epflred, transparent, false, 1px);
+  @extend .focused;
 
   .bg-dark & { color: $white; }
 
   &:hover,
   &.hover {
-    color: $link-hover-color;
     text-decoration: none;
-    border-bottom-color: $black !important;
-    .bg-dark & { border-bottom-color: $white !important; }
+    color: $link-color;
```

Comment:
shouldn't this be red?

Reply:
Not anymore according to the design updates...

Reply:
@guillaume-berry Is that right?